dspace-api/src/main/java/org/dspace/app/itemimport/ItemImport.java

```diff
@@ -66,6 +42,19 @@
 import org.w3c.dom.NodeList;
 import org.xml.sax.SAXException;
 
+import javax.mail.MessagingException;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import java.io.*;
+import java.net.URL;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
```

Comment:
All of the moving of `import` statements here is unnecessary on a maintenance branch (5.x), as it won't be brought forward into master or other branches. I'd rather we just update this to be a one-liner, as that means it better aligns with #1241 (which is the same fix for 6.x).